.github/workflows/owasp.yml

```diff
@@ -0,0 +1,111 @@
+name: OWASP ZAP Scanning
+on:
+  push:
+    branches:
+      - 'feature/owasp-scanning'
```

Comment:
@ihor-palii - I guess you were using this branch on `push` statement for testing when developing the feature, but I think, per the ticket's description, it should be to dev/* and release/* as well.